nxcomp/configure.ac

```diff
@@ -74,6 +74,30 @@ else
   CPPFLAGS="$CPPFLAGS -DIN_ADDR_T=in_addr_t"
 fi
 
+AC_ARG_WITH(info,
+            [AS_HELP_STRING([--with-info],[define -DINFO at compile time to get info session log output])])
+
+AC_ARG_WITH(valgrind,
+            [AS_HELP_STRING([--with-valgrind],[define -DVALGRIND to enable valgrind memory checker workarounds])])
+
+dnl Check whether --with-info or --without-info was given.
+
+if test "${with_info}" = yes; then
+  echo -e "enabling info output in the session log file"
```

Comment:
> echo -e is non-portable. Not your fault, I know, but please change that either to pure echo (since we're not using extended features anyway) or printf "...\n".

Now obsolete, as I use AC_ARG_ENABLE now.